### PR TITLE
Associations

### PR DESCRIPTION
**Pull request by avneetmalhotra**

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -17,4 +18,15 @@ def add_line_items_from_cart(cart)
     end
   end
 
+  def add_logged_in_user_id(user_id)
+    self.user_id = user_id unless user_id.nil?
+  end
+
+  def total_price
+    price = 0
```
Comment (nimish13) [inline app/models/order.rb]: please optimized or use better iterator

File: app/models/user.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 class User < ApplicationRecord
+  has_many :orders
```
Comment (nimish13) [inline app/models/user.rb]: what would happen to orders if we destroy a user

Comment (nimish13) [reply app/models/user.rb]: Dont allow to destroy users who have orders associated to them

**Comment by avneetmalhotra** [comment]
closed in favour of #5

**Pull request closed without merging**